src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -857,7 +857,7 @@ public String flushAll(FlushMode flushMode) {
 
   /**
    * Get the values of all the specified keys. If one or more keys don't exist or is not of type
-   * String, a 'nil' value is returned instead of the value of the specified key, but the operation
+   * String, a 'null value is returned instead of the value of the specified key, but the operation
```

Comment:
I think both are fine. According to the [redis doc](https://redis.io/commands/mget). It should be `nil`. In java, `null` is also make senses. BTW, `'null` should be `'null'`